website/content/en/blog/hello-world/index.md

```diff
@@ -80,6 +80,3 @@ So, what do you say? Want to give me a try? I promise I won’t disappoint.
 I can't wait to start on this journey with all of you. Together, we'll make
  the world of Kubernetes a better, safer, and more connected place.
  Happy cluster linking! 🚀
-
```

Comment:
I think we should keep the footnote until there's a release. We can change the wording as needed.
For example[^1] or [^2]: 
[^1]: While normal access control policies work, the implementation of privileged policy tier
 was ongoing and enabled shortly after the 0.1.0 release - it is currently part of `main` branch.
[^2]: the implementation of privileged policy tier was added in the 0.2.0 release